File: HealthCatalyst.Interfaces/Models/IPerson.cs
using System.Collections.Generic;

namespace HealthCatalyst.Interfaces.Models
{
    public interface IPerson
    {
        ICollection<IAddress> Addresses { get; set; }
        string GivenName { get; set; }
        ICollection<IInterest> Interests { get; set; }
        int PersonId { get; set; }
        ICollection<IAndroid> Phones { get; set; }
        byte[] Photo { get; set; }
        string Surname { get; set; }
    }
}

File: HealthCatalyst.Web/ClientApp/src/app/interfaces/iperson.ts
import { IAddress } from './iaddress';
import { IAndriod } from './iandriod';
import { IInterest } from './iinterest';

export interface IPerson {
    personId: number;
    givenName: string;
    surname: string;
    photo: [];
    addresses: IAddress[];
    phones: IAndriod[];
    interests: IInterest[];

    photoToSrc(): string;
}
